charts/k8s-image-swapper/Chart.yaml

```diff
@@ -13,9 +13,11 @@ keywords:
 maintainers:
   - email: enrico.stahn@gmail.com
     name: estahn
+  - email: tcyran@gmail.com
```

Comment:
@tcyran Typically there is a distinction between maintainers and contributors. This table is showing only maintainers, not contributors. Could you remove your name here? You will be mentioned in the release notes.

Reply:
Clear, removed